### DISCUSSION TITLE
Where is the current source code for gotrue-dart 0.0.5?

### DISCUSSION THREAD

**Discussion by nstrelow** [supabase/supabase#2030] [Questions] [answered]
Hey I was wondering were the current source code for the newest version is?

Thanks a lot ;D

Linking @phamhieu since you are often publishing the release version. Thanks for the new update in any case 😁 I am just curious to understand all the little changes.

**Comment by phamhieu** [comment, marked as answer]
You can checkout the repo here. https://github.com/supabase/gotrue-dart

**Reply by nstrelow** [reply]
I am not seeing the newest code for version 0.0.5. Last commit on master branch ist "chore: publish version 0.0.4"
https://github.com/supabase/gotrue-dart/commit/fa950f1cd7a304833c9ed76393ca2e9b8bdd7a45

**Reply by phamhieu** [reply]
😂 sorry, can you check again. I just pushed the code.

**Reply by nstrelow** [reply]
Thanks a lot. Kudos on the AuthOptions stuff 😁
